shared/common-adapters/markdown/react.tsx

```diff
@@ -401,16 +401,7 @@ const bigEmojiOutput: SimpleMarkdown.Output<any> = SimpleMarkdown.outputFor(
         node: SimpleMarkdown.SingleASTNode,
         output: SimpleMarkdown.ReactOutput,
         state: SimpleMarkdown.State
-      ) => (
```

Comment:
this fixes big emoji measuring getting messed up sometimes and it served no ultimate purpose